2-mandatory/5-eligible-students.js

```diff
@@ -7,7 +7,13 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
 */
 
-function getEligibleStudents() {}
+function getEligibleStudents(studentArray) {
+  studentArray.map( student => {
```

Comment:
What happened when you tested your code?

It's a good idea to comment with questions and get code review as soon as possible when you hit these problems. It's normal to encounter problems like these. Testing your code and getting code review will clear up these blockers quickly.